### DISCUSSION TITLE
How to make an Interactive Doughnut Chart with ggplot properly?

### DISCUSSION THREAD

**Discussion by Nurfalah123** [teamalgoritma/community#97] [Q&A] [answered]
I have a problem, how to create a good doughnut chart properly

plot6 <- ggplot(data = skul, mapping = aes(x = count, 
                                             y = reorder(Device, count),
                                             text = label)) +
  geom_col(mapping = aes(fill = Device), position = "stack") +
  coord_polar(theta = "x")+
  labs(x = NULL, 
       y = NULL,
       fill = NULL,
       title = "Total Penggunaan Perangkat di Universitas") +
  theme_minimal()

plot6

![donut](https://user-images.githubusercontent.com/103617918/168031085-396e4074-c7ac-425c-9535-ba2a2167754d.PNG)

**Comment by inytss** [comment, marked as answer]
Dear @Nurfalah123 , 
We have found the article related to your question, please check this out https://r-graph-gallery.com/128-ring-or-donut-plot.html.
Hope this will help :)

**Reply by Nurfalah123** [reply]
I have checked this website, however the appearance still the same. In the other hand, this is not interactive doughnut chart. So, I prefer use plotly rather than ggplot.

**Reply by inytss** [reply]
I see. If you still need to make an interactive plot using plotly, this slides maybe will help you to make your doughnut chart https://rstudio-pubs-static.s3.amazonaws.com/322072_3666b7967254432a894a72e15854597b.html#1.
Hope this will help :)

**Reply by Nurfalah123** [reply]
thank you so much for your help
